src/concurrent/atomic64_gcc_x86_64.h

```diff
@@ -0,0 +1,77 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
```

Comment:
clang should be fine anyway. The same checks for platform and compiler of gcc work with clang, thankfully.